config/config.go

```diff
@@ -281,5 +281,27 @@ func migrateConfigDir() error {
 	// ensure ~/.config exists
 	os.MkdirAll(path.Dir(DefaultConfigDirPath), 0700)
 
-	return os.Rename(LegacyConfigDirPath, DefaultConfigDirPath)
+	err = os.Rename(LegacyConfigDirPath, DefaultConfigDirPath)
+	if err != nil {
+		return err
+	}
+
+	// adapt certificate paths in kubeconfig
+	if len(KubeConfigPaths) > 0 {
+		// we only adapt the first file found (on purpose)
+		if stat, err := os.Stat(KubeConfigPaths[0]); err == nil {
```

Comment:
I don't like the `KubeConfigPaths[0]` I'm not sure what this is and makes the code hard to read. I'd also prefer to see a variable as its used multiple times.